doc/glossary.rst

```diff
@@ -1635,6 +1639,15 @@ functions or non-estimator constructors.
         full dataset. For classification, all data in a sequence of
         ``warm_start`` calls to ``fit`` must include samples from each class.
 
+    ``working_memory``
+
+        The optimal size of temporary arrays used by some algoritms.
+        Some calculations when implemented using standard numpy vectorized
+        operations involve using a large amount of temporary memory.
+        Where computations can be performed in fixed-memory chunks the user is
+        allowed to hint at the maximum size of this working memory (defaulting
+        to 1GB). 
```

Comment:
I don't think we use this outside of `set_config` and `pairwise_distances_chunked`? I don't know if we want to add it to the glossary.

If we do add it, we would need to reference the glossary from those docstrings, but then the behavior is not consistent as in functions outside of `set_config`

> When None (default), the value of ``sklearn.get_config()['working_memory']`` is used.

So postponing this until we have more occurrences might be simplest.

Reply:
Sorry, not sure to understand... Anyway I have added ```working_memory``` in the glossary because of the warning
```
scikit-learn/doc/modules/computing.rst:568: WARNING: term not in glossary: working_memory
```

Reply:
After re-reading... understood. Let's do the other way around. I have removed the link to `working_memory` and deleted `working_memory` from the glossary (bdd700f9a48bde593c1da8bad2e39879a66ab438)